### ISSUE TITLE
VERSION_PATTERN doesn't match version in Ubuntu

### ISSUE THREAD

**Issue by ak-coram** [impossibl/pgjdbc-ng#385]
Hi,

`VERSION_PATTERN` in `com.impossibl.postgres.system.Version` currently
doesn't match the database version shipped with Ubuntu 16.04:

```
  com.impossibl.postgres.system.Version.parse Version.java: 52
  java.lang.IllegalArgumentException: Invalid version string: 10.6 (Ubuntu 10.6-1.pgdg16.04+1)
  java.io.IOException: java.lang.IllegalArgumentException: Invalid version string: 10.6 (Ubuntu 10.6-1.pgdg16.04+1)
  com.impossibl.postgres.jdbc.PGSQLSimpleException: Connection Error: java.lang.IllegalArgumentException: Invalid version string: 10.6 (Ubuntu 10.6-1.pgdg16.04+1)
  SQLState: "8001"
  errorCode: 0
```

This means that the driver currently can't be used on this
platform. I'm guessing this has to do with the recent change of adding
support for tags in the version string. Could the regular expression
be made more lenient regarding what it accepts as a tag? Thanks!

**Comment by kdubb** [comment]
This fix is available in `0.9-SNAPSHOT` & `0.8.2-SNAPSHOT`

**Comment by ak-coram** [comment]
Thank you @kdubb! The fix looks good to me.
